README.md

```diff
@@ -64,7 +64,26 @@ The plugin behavior can be controlled with the following list of settings:
 
 - `enable_devicetype-library`: If enabled the [device type](https://github.com/netbox-community/devicetype-library) git repository will be automatically added for you.
 
-After updating nautobot_config.py, you will need to run `nautobot-server migrate` and then reload the nautobot service and the nautobot-worker service.
+After updating nautobot_config.py, you will need to run `nautobot-server migrate` and then reload the nautobot service and the nautobot-worker service as shown below.
+
+### Final Configuration Steps
+
+After installing the plugin and modifying `nautobot_config.py`, as the `nautobot` user, run the server migration:
+
+```no-highlight
+$ nautobot-server migrate
```

Comment:
We should standardize on running `nautobot-server post_upgrade` (which includes `migrate`, `collectstatic`, and several others that are generally good practice to run after upgrading or installing a plugin).

Reply:
That is a good idea. We should add that to the Nautobot docs as well.  I got bit by missing 'collectstatic' in another plugin.

Reply:
I'm actually working on a pretty substantial update to the core plugin docs as part of https://github.com/nautobot/nautobot/issues/13 - we should also make sure that we update all of our existing plugins' individual documentation as well though, agreed.

Reply:
Sounds good, I will update to this. I was copying what had been done previously on the Device Onboarding.